flink-end-to-end-tests/test-scripts/common.sh

```diff
@@ -296,7 +296,7 @@ function relocate_rocksdb_logs {
   # After FLINK-24785, RocksDB's log would be created under Flink's log directory by default,
   # this would make e2e tests' artifacts containing too many log files.
   # As RocksDB's log would not help much in e2e tests, move the location back to its own folder.
-  set_config_key "state.backend.rocksdb.log.dir" "/dev/null"
+  set_config_key "state.backend.rocksdb.log.dir" "/tmp"
```

Comment:
I'm a little worried that this would lead to many files in "/tmp" and disk space run out in CI machine. Is there any way we disable the log?

Reply:
I think we can set `state.backend.rocksdb.log.level` to `HEADER_LEVEL`, in that case RocksDB Log file will be very small

Reply:
But it still creates the log file, right? So `/tmp` will still accumulate trash log files. How about setting log level to `header` and keeping them in Flink's log directory?

Reply:
> But it still creates the log file, right? So `/tmp` will still accumulate trash log files. How about setting log level to `header` and keeping them in Flink's log directory?

yeah, that's what I mean !

Reply:
@Zakelly I've updated the PR and the CI has passed now, can you take a look